tests/test_matrix_props/test_is_permutation.py

```diff
@@ -0,0 +1,42 @@
+"""Test is_permutation."""
+import numpy as np
+
+from toqito.matrix_props.is_permutation import is_permutation
+
+def test_is_simple_permutation():

```

Comment:
These functions are not part of a class--therefore, PEP-8 requires two spaces between each function definition.

Reply:
OK thanks so much.  I am new to PEP-8.  Will modify to follow that standard.